zuul.d/jobs.yaml

```diff
@@ -2173,7 +2173,7 @@
       - Project:apache/hbase
       - Application:Hbase@master
       - OS:ubuntu-xenial
-      - Arch:aarch64
+      - Arch:arm64
       - BuildType:Compile
```

Comment:
looks duplicated with https://github.com/theopenlab/openlab-zuul-jobs/pull/588

Reply:
Maybe we remain 'aarch64' for Arch metadata? We specify the 'Arch' comes from uname -a in redesign doc, or modify the doc to just tell we use arm64?

Reply:
@huangtianhua I have updated all of the related documents, include: redesign and docs.openlabtesting.org, please doube check it, thank you. And I think `Arch:arm64` is clear enough.

Reply:
OK, let's set it to arm64, and I will change related in my spark job.